### ISSUE TITLE
Support textures

### ISSUE THREAD

**Issue by bopoh13** [kovacsv/Online3DViewer#22]
Hello Viktor!

Do you plan to support textures in the future?

- dds: https://p3dm.ru/files/auto/tanks/14194-is-2.html
- tga: https://p3dm.ru/files/auto/tanks/13814-t-34-85.html

**Comment by kovacsv** [comment]
Hello!

Online3DViewer supports textures (at least the common file formats, like jpg, png, or gif). You have to open or drag and drop them along with the model file to make it work. I'm not sure about dds and tga format textures.

**Comment by bopoh13** [comment]
Yes, in other models, textures in the format of pictures are displayed. And the above models look something like this: https://p3d.in/L7sdw/clean :eyes:
